src/js/controller.js

```diff
@@ -175,7 +173,7 @@ export const MediaUIStates = {
 
       // Account for cases where metadata from slotted media has an "empty" seekable (CJP)
       if (!start && !end) return undefined;
-      return [Number(start.toFixed(3)), Number(end.toFixed(3))].join(':');
+      return [Number(start.toFixed(3)), Number(end.toFixed(3))];
```

Comment:
was going back and forth on this: Should we truncate the values here or keep the precision for "real" values and only truncate on serialization? I leaned in this direction, only because our media state receiver component state is stored via attrs anyway (e.g. `set mediaSeekable` -> setAttribute('mediaseekable', serializedVal)` to store the value), so unless/until we move to private vars for state "source of truth", this felt like the right approach.

Reply:
I agree, 3 decimals precision seems sufficient

Reply:
Broadly, I'd lean towards not changing our decisions on data modeling in this PR, but I'd also lean towards keeping the data simple for now. @AdamJaggard and I discussed data shape modeling for this during his component props add and he settled on an easier to work with API ("tuples"). When we first introduced seekable, we decided to keep it simple/consistent with the way it works for `HTMLMediaElement` unless/until it makes sense to support more complex cases, which would likely tie into some kind of media-ui-extension.

Reply:
Makes sense 👍